### PR TITLE
Distribute points question reverts to 100 #5924

### LINKED ISSUE #5924: Distribute points question reverts to 100
Received from user and confirmed as bug.
```
I am trying to set up a session and am running into a glitch. I have added a distribute points (among recipients) question. Every time I try change the total points to distribute to 25 x (number of recipients) when I click save changes, it reverts back to the 100 x (number of recipients). ?
```

### PR DESCRIPTION
**Pull request by chowyb**
Fixes #5924

Changes the relevant input boxes to use a different name and id, and added functions to do frontend validation and backend processing of the input.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@chowyb you should not include the first two commits, otherwise we might as well merge to master branch and draft a new release from there.

**Comment by chowyb** [comment]
@wkurniawan07 okay, but is there any way to do this in Git directly? I'm not sure, but I'll try replicating the changes on a fresh branch in the meantime.

**Comment by wkurniawan07** [comment]
Should be straightforward with interactive rebase `git rebase -i release`? The first two commits can be `drop`-ed.

File: src/main/java/teammates/common/util/Const.java [hunk 1]
```diff
@@ -639,6 +639,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_QUESTION_CONSTSUMNUMOPTION = "constSumNumOption";
         public static final String FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION = "constSumPointsPerOption";
```
Comment (chowyb) [inline src/main/java/teammates/common/util/Const.java]: This particular constant can be quite confusing when compared to the newer ones; the constant here is used to determine if the points to be distributed is fixed, or a specified number multiplied by the number of options. As it is too risky to change this name right now, I'm leaving it as it is.

Comment (damithc) [inline src/main/java/teammates/common/util/Const.java]: Add a TODO?

**Comment by chowyb** [comment]
Ready for review; the most recent commit is not expected to cause any test failures.

**Comment by wkurniawan07** [comment]
Confirmed that the bug is resolved :+1:

File: src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java [hunk 2]
```diff
@@ -63,6 +63,8 @@ public boolean extractQuestionDetails(
         String distributeToRecipientsString = null;
         String pointsPerOptionString = null;
         String pointsString = null;
+        String pointsForEachOptionString = null;
+        String pointsForEachRecipientString = null;
         String forceUnevenDistributionString = null;
         boolean distributeToRecipients = false;
         boolean pointsPerOption = false;
```
Comment (damithc) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: These booleans are not named as booleans. Can be fixed in a separate PR.

**Comment by damithc** [comment]
One lesson to learn from this embarrassing bug is we should make our test values as diverse as possible. Using the same test values in multiple places can hide bugs like this one. @tarun14110 take note.

**Comment by tarun14110** [comment]
Apologies for that. I will keep this thing in mind.

**Comment by wkurniawan07** [comment]
- [x] Released as V5.85.01
- [x] Merged to current `master` branch

**Pull request merged**